src/core/qgsdistancearea.cpp

```diff
@@ -384,6 +400,12 @@ QgsPointXY QgsDistanceArea::computeSpheroidProject(
   double a = mSemiMajor;
   double b = mSemiMinor;
   double f = 1 / mInvFlattening;
+  if ( ( ( a < 0 ) && ( b < 0 ) ) ||
+       ( ( p1.x() < -180.0 ) || ( p1.x() > 180.0 ) || ( p1.y() < -85.05115 ) || ( p1.y() > 85.05115 ) ) )
```

Comment:
We could, but I remember double precision problems being caused that returned unexpected results (independent of the Mercator tile logic) for this area.

The mathematical problem is also the reason why there is no tile support for this area.